### PR TITLE
Add Continuous Integration article

### PR DESCRIPTION
**Pull request by m-murasovs**
Preview: https://docs.apify.com/actors/development/continuous-integration?version=revamp/cont-int-article

Describing the general process of continuous integration for actors. When the [Public actors guide](https://www.notion.so/apify/Public-actor-Guide-7c2ef51d4c9846afa33567dc516cabfa#0994f3fe3ddc4f9393a561ad3c913f7a) is integrated, I'll add links to more info for the 'ongoing testing' section

### CODE REVIEW

File: docs/actors/development/continuous_integration.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+[Find your Apify token in the Apify app](https://my.apify.com/account#/integrations).
+
+![Apify token in app]({{@asset actors/development/images/ci-token.png}})
+
+[Add the token to GitHub secrets](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository). Go to **your repo > Settings > Secrets > New repository secret**.
+
+Add the [**build actor** API endpoint URL](/api/v2#/reference/actors/build-collection/build-actor) to GitHub secrets. Configure it to use your actor's ID and your API token.
+
+![Add build actor URL to secrets]({{@asset actors/development/images/ci-add-build-url.png}})
```
Comment (mtrunkat) [inline docs/actors/development/continuous_integration.md:80]: I'd also add the URL in a code block below to make it copyable

File: docs/actors/development/continuous_integration.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+
+This article will focus on GitHub but [we also have a guide for Bitbucket](https://help.apify.com/en/articles/1861038-setting-up-continuous-integration-for-apify-actors-on-bitbucket).
+
+## TL;DR
+
+Below is an example GitHub Actions workflow that will run your tests and build your actor every time you push your code to GitHub. This workflow supports both **latest** and **beta** builds. Copy the code into **separate files** in your actor repo's **.github/workflows** directory: e.g. **.github/workflows/main.yml** and **.github/workflows/beta.yml**.
+
+Each time you push to the **main/master** branch, a new **latest** version of your actor will be built. When you push to **develop**, a **beta** version will be built.
+
+```marked-tabs
```
Comment (mtrunkat) [inline docs/actors/development/continuous_integration.md:22]: I am not 100% sure about naming main and beta, what about  to name it by branches main and develop or latest and beta

**Review by mtrunkat — CHANGES_REQUESTED** [review]

File: docs/actors/development/continuous_integration.md (hunk 2 above)
Comment (m-murasovs) [reply docs/actors/development/continuous_integration.md:22]: Do you mean like this?
```
Each time you push to the **main** branch, a new **main** version of your actor will be built. When you push to **develop**, a **develop** version will be built.
```

Comment (mtrunkat) [reply docs/actors/development/continuous_integration.md:22]: I mean the filenames in workflows and multi-file editor.

Comment (m-murasovs) [reply docs/actors/development/continuous_integration.md:22]: Ah, I see! Sure thing!

**Review by mtrunkat — APPROVED** [review]

**Pull request merged**